app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Place.java

```diff
@@ -10,7 +10,7 @@
 
     private String id;
 
-    private String place_id;
+    private String placeId;
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.